test/hotspot/jtreg/compiler/loopopts/superword/SumRedAbsNeg_Double.java

```diff
@@ -110,6 +90,11 @@ public static void sumReductionInit(
         }
     }
 
+    /* Does not vectorize due to profitability heuristic
+       (with or without store) in SuperWord::profitable. */
+    @Test
+    @IR(applyIfOr = {"SuperWordReductions", "false", "LoopMaxUnroll", "< 8"},
+        failOn = {IRNode.ADD_REDUCTION_VD, IRNode.ABS_V, IRNode.NEG_V})
```

Comment:
@danielogh Are you sure that this would not vectorize if `sve` is available? It may not fail on any Oracle test machines, but we only have `x64` and `aarch64` (only have `asimd`, not `sve`). Have you ever let this be tested explicitly with `sve` support?

Reply:
@fg1417 could you please run this patch on a machine with `sve` support? It would be good to know that all IR rules are ok.

Reply:
> @fg1417 could you please run this patch on a machine with `sve` support? It would be good to know that all IR rules are ok.

Sure. I'll do the testing on our sve machine. Thanks.